src/constants/homeLinks.ts

```diff
@@ -0,0 +1,38 @@
+export const HOME_LINKS = [
```

Comment:
this could be in ..../constants/links.ts in my opinion

Reply:
+1